src/sidebar/components/focused-mode-header.js

```diff
@@ -4,6 +4,11 @@ const { createElement } = require('preact');
 
 const useStore = require('../store/use-store');
 
+/**
+ * Render a control to interact with any focused "mode" in the sidebar.
+ * Currently only a user-focus mode is supported but this could be broadened
+ * and abstracted if needed. Allow user to toggle in and out of the focus "mode."
```

Comment:
"Allow[s] user to ..."

Reply:
It's in the imperative mood, so `Allow` is technically correct here.